common/chat/conversation.ts

```diff
@@ -15,6 +15,11 @@ const TALKJS_API_URL = `https://api.talkjs.com/v1/${process.env.TALKJS_APP_ID}`;
 const TALKJS_CONVERSATION_API_URL = `${TALKJS_API_URL}/conversations`;
 const TALKJS_API_KEY = process.env.TALKJS_API_KEY;
 
+const ONE_ON_ONE_SYSTEM_MESSAGE =
+    'Standardmäßig werdet ihr über neue Chat-Nachrichten per E-Mail informiert. In den Einstellungen könnt ihr eure Benachrichtigungen anpassen. Wir erwarten, dass alle Chat-Nachrichten respektvoll sind. Falls es Beleidigungen oder andere Probleme gibt, meldet euch bitte unter sorgen-eule@lern-fair.de bei uns.';
+const GROUP_CHAT_SYSTEM_MESSAGE =
+    'In diesem Gruppen-Chat können Kursleiter:innen wichtige Ankündigungen und weitere Informationen zum ihrem Kurs mit allen Schüler:innen teilen. Dabei können Kursleiter:innen einstellen, ob auch Schüler:innen in den Gruppen-Chat schreiben dürfen. Der Chat wird 30 Tage nach Ende des Kurses inaktiv. Wir erwarten, dass alle Chat-Nachrichten respektvoll sind. Falls es Beleidigungen oder andere Probleme gibt, meldet euch bitte unter sorgen-eule@lern-fair.de bei uns.';
+
```

Comment:
Can we localize that, with an object? So if other languages will be used in the future, we at least serve a concept. 
const ONE_ON_ONE_SYSTEM_MESSAGE = {
  de: "...."
};

Do we have any other options for localization? @Jonasdoubleyou

Reply:
We could put this in another file `chat/localization.ts`, as there are some more messages to come.

Reply:
Having some "i18n like file" sounds good to me :)